vclib-aries/build.gradle.kts

```diff
@@ -13,26 +13,31 @@ val artifactVersion: String by extra
 group = "at.asitplus.wallet"
 version = artifactVersion
 
+val iosDisabled: String? by extra
```

Comment:
Why?

Reply:
to work around an IDEA bug. it is now documented in DEVELOPMENT.md